doc/source/india-wins-bid-hosting-chess-olympiad.md

```diff
@@ -0,0 +1,19 @@
+# 🏰 India wins bid for hosting Chess Olympiad, all set to host most awaited chess event
+
+The 44th FIDE Chess Olympiad will be held in Chennai, Tamil Nadu, India. This news was announced at a press conference in Chennai on Wednesday, after the FIDE Council had announced that India won bid the night before. The event was originally to be held in Russia but the war in Ukraine made it impossible. The Olympiad will take place in July/August, \"not very far off from the dates originally planned,\" FIDE said. The original dates were 26 July-August 8, 2022.
+
+<br>It will be the first time ever that India will host Chess Olympiad. Bharat Singh Chauhan, the secretary of the All Indian Chess Federation(AICF) and the tournament director, called it a \"historic press conference\" and \"big moment for all of us.\" He talked about how fast everything happened, from the moment FIDE decided to cancel Moscow until Chennai won the bid. \"It was very quick. We met the Tamil Nadu's CM, M. K. Stalin, and within five minutes he agreed.\" The official approval letter was sent in a few hours later, and on the very same day, 1,200 rooms were booked in hotels in Mamallapuram. \"Now we have booked nearly 3,000 rooms, so things are moving very quickly,\" Bharat Singh Exclaimed. The venue will be the Four Points by Sheraton Mahabalipuram Resort & Convention Center, 60 kilometers south from central Chennai. It is located in Mamallapuram, one of the famous tourist sites in India and a UNESCO World Heritage Site of seventh- and eighth-century Hindu monuments.
```

Comment:
```suggestion
<br>It will be the first time ever that India will host Chess Olympiad. Bharat Singh Chauhan, the secretary of the All Indian Chess Federation(AICF) and the tournament director, called it a \"historic press conference\" and \"big moment for all of us.\" He talked about how fast everything happened, from the moment FIDE decided to cancel Moscow until Chennai won the bid. \"It was very quick. We met the Tamil Nadu's CM, M. K. Stalin, and within five minutes he agreed.\" The official approval letter was sent in a few hours later, and on the very same day, 1,200 rooms were booked in hotels in Mamallapuram. \"Now we have booked nearly 3,000 rooms, so things are moving very quickly,\" Bharat Singh exclaimed. The venue will be the Four Points by Sheraton Mahabalipuram Resort & Convention Center, 60 kilometers south from central Chennai. It is located in Mamallapuram, one of the famous tourist sites in India and a UNESCO World Heritage Site of seventh- and eighth-century Hindu monuments.
```